doc/en/user/docs/community/features-templating/directives.md

```diff
@@ -12,49 +12,31 @@ The following constitutes a summary of all the template directives and it is mea
-  **Usage**                                                            **Syntax**                           **Description**
-
-  property interpolation                                               \${property}                         specify it as an attribute value (`"json_attribute":"${property}"`)
-
-  cql evaluation                                                       \$\${cql}                            specify it as an element value (`"json_attribute":"$${cql}"`)
-
-  setting the evaluation context for child attributes.                 \${source}.                          specify it as the first nested object in arrays (`{"$source":"property"}`) or as an attribute in objects (`"$source":"property"`)
-
-  filter the array, object, attribute                                  \$filter                             specify it inside the first nested object in arrays (`{"$filter":"condition"}`) or as an attribute in objects (`"$filter":"condition"`) or in an attribute next to the attribute value separated by a `,` (`"attribute":"$filter{condition}, ${property}"`)
-
-  defines options to customize the output outside of a feature scope   \$options                            specify it at the top of the JSON template as a JSON object (GeoJSON options: `"$options":{"flat_output":true, "separator":"."}`; JSON-LD options: `"$options":{"@context": "the context json", "encode_as_string": true, "@type":"schema:SpecialAnnouncement", "collection_name":"customCollectionName"}`).
-
-  allows including a template into another                             \$include, \$includeFlat             specify the `$include` option as an attribute value (`"attribute":"$include{subProperty.json}"`) and the `$includeFlat` as an attribute name with the included template path as a value (`"$includeFlat":"included.json"`)
-
-  allows a template to extend another template                         \$merge                              specify the `$merge` directive as an attribute name containing the path to the extended template (:code: ``"\$merged":"base_template.json"``).
-
-  allows null values to be encoded. default is not encoded.            \${property}! or \$\${expression}!   ! at the end of a property interpolation or cql directive (`"attribute":"${property}!"` or `"attribute":"$${expression}!"`).
-  -------------------------------------------------------------------- ------------------------------------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| **Usage** | **Syntax** | **Description** |
+|-|-|-|
+|property interpolation | \${property} | specify it as an attribute value (`"json_attribute":"${property}"`) |
+| cql evaluation | \$\${cql} | specify it as an element value (`"json_attribute":"$${cql}"`) |
```

Comment:
I wish I could tell what you were changing there, checking the rst conversion I am finding that these were intended to be code examples and are getting \\$ shown in the output.